request-comply-advantage/README.md

```diff
@@ -0,0 +1,424 @@
+# Request Comply Advantage Blacklist
+This README contains a brief description of [the example content](./index.js)
+
+## What exactly is Comply Advantage?
+Comply Advantage is a tech company that creates a solution detecting and reducing fraud losses. The Comply Advantage focus is to identify and reduce the risks associated with illegal financial activities.
+
+Comply Advantage uses advanced technologies like automatic learning and natural language process to analyze big financial data and detect abnormal or suspicious patterns. Their platform combines data from different sources, such as sanctions lists, regulatory reports, news, and social media, to give its clients a comprehensive view of potential risks.
+
+## How can I use Comply Advantage?
+
+### Create your moffin account
+First you will need an account in the Moffin Sandbox
+environment. You can create one easily in this
+[page](https://moffin.mx/sign_up).
+
+### Generate an Access Token
+Go to the configuration panel in moffin to create an *access Token*.
+
+Go to "Configuración" > "API" > "Crear nuevo token":
+* [Configure token in sandbox](https://sandbox.moffin.mx/configuracion/api)
+* [Configure token in production](https://app.moffin.mx/configuracion/api)
+
+Then you can configure the token as is show in this example.
+```js
+const client = Axios.create({
+  baseURL: 'https://sandbox.moffin.mx/api/v1',
+  headers: {
+  'Authorization': 'Token ' + process.env.API_TOKEN
+  }
+})
+```
+
+You can find more information of which information you can send to the API in our [docs](https://moffin.mx/docs).
+
+### Comply Advantage Globals
+
+You can configure the level of matching a profile _Fuzziness_ the accepted value can be 0 to 1.0 it could be until 2 decimals.
```

Comment:
Creo que añadiría una pausa o punto luego de Fuzziness aqui.

Tambien: *the accepted value can be between 0 and 1.0, up to 2 decimals.